src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -169,10 +188,11 @@ protected void processFiltered(File file, FileText fileText) throws CheckstyleEx
         if (CommonUtils.matchesFileExtension(file, getFileExtensions())) {
             final String msg = "%s occurred during the analysis of file %s.";
             final String fileName = file.getPath();
+            final FileContents contents = new FileContents(fileText);
+
             try {
                 if (!ordinaryChecks.isEmpty()
```

Comment:
if ordinaryChecks and  commentChecks are empty we should not do filtering as there will be no messages. Filtering should be inside if block.